wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenItem.tsx

```diff
@@ -44,9 +48,14 @@ function TokenItem(props: TokenItemProps) {
   const { classes } = useStyles();
   const theme = useTheme();
 
-  const chainConfig: ChainConfig | undefined =
-    config.chains[props.token.tokenId?.chain ?? ''];
-  const address = props.token.tokenId?.address;
+  const { chain, token } = props;
+  const chainConfig: ChainConfig | undefined = config.chains[chain];
+  // If the token is native to the chain, show the token's address.
+  // Otherwise, show the wrapped token's address.
+  const address =
+    chain === token.nativeChain
+      ? token.tokenId?.address
+      : getWrappedToken(token)?.foreignAssets?.[chain]?.address;
   const explorerURL = `${chainConfig?.explorerUrl}address/${address}`;
```

Comment:
If `address` is eval'ed to `undefined` above, should `explorerURL` also be `undefined`?

Reply:
changed